packages/cli/src/utils/resolver.ts

```diff
@@ -21,38 +22,20 @@ export const serviceResolver = async (api: IApi, sidOrHash: hash | string): Prom
   }
 }
 
-export const instanceResolver = async (api: IApi, sidOrHash: hash | string): Promise<hash> => {
+export const instanceResolver = async (api: LCD, sidOrHash: string): Promise<string> => {
```

Comment:
i got the following error:
```
➜  js-sdk git:(feature/lcd-server) ./packages/cli/bin/run service:list
HASH                                         SID                 INSTANCES                                    RUNNERS
2msvJE8est3tTfos2wEnSQMW9oRVjUtTGDoSeyZ4x53G test-service        EGyLyEVdxS5oseAUwRtrZteH9RhG1dzmR5zqimXrzdtX CYAJSYuy1JFgC6Cp88tmLXh4j518oiJtC2qRfoRhta4C
6q7pBFYgStFdYpVgBcAN6wUcgtc97hevwvtcHpCSiMr1 emit-event-interval ELhRBQd3tCWmTUqPFYSWEbkgs1TikkCgHc1YBjTdgK2P D6W1QEynLygnRftEhdHGyZqtvgbzFg21VVN5AXwPTikY
➜  js-sdk git:(feature/lcd-server) ./packages/cli/bin/run service:stop test-service
Stop running services... !
 ›   Error: multiple instances for the service test-service
```
not sure it's related to this PR...

Reply:
fixed, this was an issue with the API that doesn't support the filters anymore. Filtering on the client side now.